docs/ref/models/expressions.txt

```diff
@@ -37,6 +37,13 @@ determine the result's field type, such as complex expressions that mix field
 types. For example, adding a ``DecimalField()`` and a ``FloatField()`` requires
 an output field, like ``output_field=FloatField()``.
 
+``output_field`` is also useful for encapsulating type conversion behavior in
```

Comment:
There is probably a simpler wording here than "encapsulating". Maybe:

``output_field`` also allows using custom fields that perform type conversions
outside a specific model field context.

or similar